client-sdk/go/types/transaction.go

```diff
@@ -184,8 +184,9 @@ func (cr *CallResult) IsSuccess() bool {
 
 // FailedCallResult is a failed call result.
 type FailedCallResult struct {
-	Module string `json:"module"`
-	Code   uint32 `json:"code"`
+	Module  string `json:"module"`
+	Code    uint32 `json:"code"`
+	Message string `json:"message"`
```

Comment:
The reason why there was no message field before was to reduce the output size. I guess we could at least make it optional (`omitempty` here).

Reply:
ok